Cli/Basic.lean

```diff
@@ -232,6 +232,17 @@ section Configuration
       | "" => none
       | s  => s.toInt?
 
+  open Lean System in
+  /-- A custom command-line argument parser that allows either relative paths to Lean files,
+  (e.g. `Mathlib/Topology/Basic.lean`) or the module name (e.g. `Mathlib.Topology.Basic`). -/
+  instance : ParseableType Name where
+    name     := "Name"
+    parse? s :=
+      if s.endsWith ".lean" then
+        some <| (s : FilePath).withExtension "" |>.components.foldl Name.mkStr Name.anonymous
```

Comment:
```suggestion
        let pathComponents := (s : FilePath).withExtension "" |>.components
        some <| pathComponents.foldl Name.mkStr Name.anonymous
```
?
Not a big fan of mixing `<|` and `|>` in the same line.

This should probably also account for `«...»`.

Reply:
Sorry, I'm not certain what account for `«...»` you have in mind. This code trying to parse a path: no one puts `«...»` in their file names, do they?